DataFormats/Detectors/CTP/include/DataFormatsCTP/Configuration.h

```diff
@@ -56,41 +62,46 @@ struct CTPDetector {
 struct CTPCluster {
   CTPCluster() = default;
   std::string name;
-  uint32_t detectorsMask;
-  std::vector<std::string> detectorsNames;
+  o2::detectors::DetID::mask_t detectorsMask;
+  std::vector<o2::detectors::DetID> detectors;
```

Comment:
why do you need both detectorsMask and `std::vector<o2::detectors::DetID> detectors` ? Don't they encode the same detectors list?

Reply:
ok, keeping only DEtID vector